playbooks/common/openshift-checks/install.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Health Check Checkpoint Start
+  hosts: oo_all_hosts
+  gather_facts: false
+  tasks:
+  - name: Set Health Check 'In Progress'
+    set_stats:
+      data:
+        installer_phase_health: "In Progress"
+      aggregate: false
+
+- name: OpenShift Health Checks
+  hosts: oo_all_hosts
+  any_errors_fatal: true
```

Comment:
I'm also adding `any_errors_fatal: true` here like I did for #5741

Reply:
Interesting, were any errors not fatal before?

Reply:
I confirmed this was a problem when troubleshooting for https://github.com/openshift/openshift-ansible/pull/5741